main/2020/CG-11-10.md

```diff
@@ -0,0 +1,41 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the November 10th video call of WebAssembly's Community Group
+
+- **Where**: zoom.us
+- **When**: November 10th, 4pm-5pm UTC (November 10th, 9am-10am Pacific Daylight Time)
```

Comment:
I think this will be 5pm UTC, 9am PST (DST ends on Nov 1st)

Reply:
ah, fixed (on the assumption that the PT time is fixed at 9am)